gemini/tests/set3/signposts.js

```diff
@@ -9,6 +9,10 @@ gemini.suite('signposts', (child) => {
         })
         .setCaptureElements('.content-area')
```

Comment:
I know this PR doesn't change this line but we should wrap all demos in a container like `.clr-example` and take that screenshot instead of the `.content-area`. Otherwise the gemini regression images add unnecessary height to the images because the `.content-area` is occupies the entire height thats left from the header. May be a TODO for later?

Reply:
I can make this change now. Thanks.

Reply:
I will add a TODO for later.